src/common/constants.ts

```diff
@@ -3,3 +3,4 @@ export const IMAGE_ADDRESS = 'https://raw.githubusercontent.com/cuappdev/assets/
 export const MAX_NUM_DAYS_OF_TRENDING_ARTICLES = 30;
 export const IOS = 'IOS';
 export const ANDROID = 'ANDROID';
+export const IMMEDIATELY_EXPIRE_DATE = 'December 17, 1995 03:24:00';
```

Comment:
what is this constant for? I'm not sure what is 'immediately expiring'

Reply:
Since new accounts can be made mid week, their weekly debriefs would not really be valid. But I think it will be better to make new users' weekly debriefs be `undefined` like Shungo proposed instead.